app/grants/migrations/0004_auto_20180918_2322.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.1 on 2018-09-18 23:22
+
+from django.db import migrations, models
```

Comment:
We'll want to delete all of these `grant/migrations/*.py` and recreate them under one migration.